### ISSUE TITLE
Potential bug: misusage in information entropy

### ISSUE THREAD

**Issue by Luolc** [huawei-noah/DAFL#12]
https://github.com/huawei-noah/DAFL/blob/48c2ade7f029df50009f8ca0afcd9cd9d694c2de/DAFL-train.py#L179

Here in the code you used `log10` for information entropy loss. However the cross entropy loss implemented in PyTorch uses nature log. The math op should be used consistently across different loss terms (one hot loss and cross entropy loss), otherwise there would be an unexpected `ln 10` scaling.

**Comment by HantingChen** [comment]
Thanks for your correction. In fact, the scaling can be ignored by tuning hyper-parameters of one hot loss and cross entropy loss. So it won't affect the results.

**Comment by Luolc** [comment]
Thanks for the reply.

> the scaling can be ignored by tuning hyper-parameters of one hot loss and cross entropy loss. So it won't affect the results.

Yes I know the only difference is a scaling, yet it may confuse the readers. When setting same weights to one-hot loss and information entropy loss, one would expect the two losses have same importance.

**Comment by HantingChen** [comment]
fixed and thanks.
